resources/assets/js/components/pages/admin/categories/editor/Descriptors.vue

```diff
@@ -103,10 +104,24 @@
                     icon: ""
                 });
             },
-            parseAllowedValues (descriptor) {
-                return descriptor.allowed_values.split(",").map((value) => {
+            splitTrimJoin (string, splitOn, joinWith) {
+                return string.split(splitOn).map((value) => {
                     return value.trim();
-                }).join("|");
+                }).join(joinWith);
```

Comment:
In an effort to make the allowed value parsing more expandable down the road, common functionality that Is likely to be used on future descriptors was pulled into a single method. It splits a string, trims each element, and then joins it again with the specified characters.